kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -45,6 +45,9 @@ object TypelevelKernelPlugin extends AutoPlugin {
       "Command aliases defined for this build"
     )
 
+    lazy val tlPrePRSteps =
+      settingKey[List[String]]("Steps to be performed before a user submits a PR")
```

Comment:
Mostly curious, why did you introduce this setting in the kernel plugin? I think we could introduce and set the default in the CI plugin, without need a blank default.

Also the doc can be more specific that these are the steps run by the `prePR` command alias.

Reply:
Added it to kernel because at my work we are using the Github plugin without using `TypelevelCiPlugin`, so for our use it made more sense to push it into Kernel.  Maybe there is an argument that we should be extending `TypelevelCiPlugin`... I`ll take a look next week, but  now that I`ve looked closer at the CIPlugin it might make sense.

Reply:
If you are only using the kernel plugin, then it's fairly straightforward to do whatever you want directly with `tlCommandAliases`.

Reply:
While it is, and I did... I thought I would basically push back up the changes I was making for myself internally.  My thought process was that as people add different 'linters' or other things that are expected to pass in CI, such as scalafix/fmt, mima, or sbt-explicit-dependency, or sbt-missinglink , I wanted an easy way to aggregate those into a check at the the same time (in the same auto-plugin) as i was adding the github workflow. 

What ended up happening in my world is people started defining a command alias called 'build'  in each project and just put in the various steps, but somehow it would always be out of sync with what we actually did in CI.

Reply:
Alright, so the reason I'd prefer to keep prePR in kernel is because we have our own custom ci steps and we'd like the easier to extend prePR.